server/src/entity/AskedQuestion.ts

```diff
@@ -0,0 +1,45 @@
+import { Entity, Column, PrimaryGeneratedColumn, Generated } from 'typeorm';
+
+@Entity({ name: 'qms_asked_question' })
+export class AskedQuestion {
+  @PrimaryGeneratedColumn('uuid', { name: 'id' })
+  id: string;
+
+  @Column('uuid', {
+    name: 'contest_contender_id',
+    nullable: false,
+  })
+  contestContenderId: string;
+
+  @Column('uuid', {
+    name: 'question_id',
+    nullable: false,
+  })
+  questionId: string;
```

Comment:
Questions could be dynamic, in that case, a question will look like string template (with parameters):
`What is {{firstNumber=random}} plus {{secondNumber=random}}?`
that's why we probably should store here actual question and answer:
`What is 3 plus 2?`
`5`
besides the question id.

Please contact @mrproper1337, he working on this now.

Reply:
Let's go with a minimalistic approach - store **questionId** (just for reference), **generatedQuestion**, **generatedAnswer** (which in case of static question can be duplicated, and in case of dynamic - generated with stored in _Question_ instance method)

Reply:
We can sync up on a call at any suitable time, to ensure that everyone is on the same page if smt isn't clear

Reply:
Done